weasyl/character.py

```diff
@@ -186,60 +190,115 @@ def is_hidden(charid):
     return bool(results and results[0][0])
 
 
-def select_view(userid, charid, rating, ignore=True, anyway=None):
-    query = define.execute("""
+def _select_character_and_check(userid, charid, rating=None, ignore=True, anyway=False, increment_views=True):
+    """Selects a character, after checking if the user is authorized, etc.
+
+    Args:
+        userid (int): Currently authenticating user ID.
+        charid (int): Character ID to fetch.
+        rating (int): Maximum rating to display.
+        ignore (bool): If should respect ignored or blocked tags.
```

Comment:
Why 'if should' over 'Whether to'? I don't have a problem with it. It's just not something I've seen before.